### PR TITLE
Test cases for client

### PR DESCRIPTION
**Pull request by Deepika1095**
## Description

Describe the changes made and why they were made.

Ignore if these details are present on the associated [Apache Fineract JIRA ticket](https://github.com/apache/fineract/pull/1284).

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Write the commit message as per https://github.com/apache/fineract/#pull-requests

- [ ] Acknowledge that we will not review PRs that are not passing the build _("green")_ - it is your responsibility to get a proposed PR to pass the build, not primarily the project's maintainers.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Follow coding conventions at https://cwiki.apache.org/confluence/display/FINERACT/Coding+Conventions.

- [ ] Add required Swagger annotation and update API documentation at fineract-provider/src/main/resources/static/legacy-docs/apiLive.htm with details of any API changes

- [ ] Submission is not a "code dump".  (Large changes can be made "in repository" via a branch.  Ask on the developer mailing list for guidance, if required.)

FYI our guidelines for code reviews are at https://cwiki.apache.org/confluence/display/FINERACT/Code+Review+Guide.

### CODE REVIEW

**Review by MageziArthur — DISMISSED** [review]
LGTM

**Review by MageziArthur — DISMISSED** [review]
LGTM

File: karate-tests/src/test/resources/features/portfolio/clients/clientcreation.feature [hunk 1]
```diff
@@ -3,3 +3,43 @@ Feature: Test client apis
...
+    * def legalFormId = 2
+    * def result = call read('classpath:features/portfolio/clients/clientsteps.feature@findbyclientid') { clientId : '#(createdClientId)'}
+    * def client = result.client
+    * match createdClientId == client.id
+    * match legalFormId == client.legalForm.id
+
+    #update fetched client
+    * def accountNo = client.accountNo
+    * def updatedClient = call read('classpath:features/portfolio/clients/clientsteps.feature@updateEntityStep') { clientId : '#(createdClientId)', accountNo : '#(accountNo)'}
+    * assert createdClientId == updatedClient.res.resourceId
```
Comment (hero78) [inline karate-tests/src/test/resources/features/portfolio/clients/clientcreation.feature:28]: Could you assert the params updated instead of the clientId

**Review by hero78 — COMMENTED** [review]
We need more assert on params created and updated

File: karate-tests/src/test/resources/features/portfolio/clients/clientcreation.feature (hunk 1 above)
Comment (Deepika1095) [reply karate-tests/src/test/resources/features/portfolio/clients/clientcreation.feature:28]: Okay I will do that

Comment (Deepika1095) [reply karate-tests/src/test/resources/features/portfolio/clients/clientcreation.feature:28]: I made changes please review again

**Review by hero78 — APPROVED** [review]
LGTM

**Review by MageziArthur — APPROVED** [review]
LGTM

**Pull request merged**